cli/azd/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 0.3.0-beta.5 (2022-10-21)
+
+### Bugs Fixed
+
+- [[#979]](https://github.com/Azure/azure-dev/pull/979) Fix provisioning template with non string outputs.
```

Comment:
Is this correct or was it a copy/paste from the beta.4 build?

Reply:
both :)  . It is correct and copy-pasted.

The beta.4 was created to fix this issue, but ended up bringing a new bug.

beta.5 is beta.4 minus the bug introduced on beta.4 (deprecated).  beta.5 was created from a buxfix/ branch, having only the cherry pick for the fix